readme.md

```diff
@@ -5,10 +5,23 @@ go-nitro
 
 <p align="center">Implementation of the <a href="https://docs.statechannels.org">Nitro State Channels Framework</a> in Golang and Solidity.</p>
 
-
 ## Usage
 
-Go-nitro is work-in-progress library code with an evolving API.
+> ⚠️ Go-nitro is pre-production software ⚠️
+
+### As a Service
+
+Go-nitro can be run as a system service with an RPC api. Go-nitro is configured to seek out a local hardhat blockchain with some specific configuration. The configured hardhat node is available here: https://github.com/statechannels/hardhat-docker.
+
+After the hardhat docker node is running, go-nitro can be started from the root directory with
+
+```
+go run ./...
```

Comment:
Is this meaningfully different to `go run .`?

* be good to give a sense of the arguments which may be passed in, or at least point to where that information is (`go run . -help`).
* let's also point to the `go build` way of doing it, too?